build_simulators.sh

```diff
@@ -10,9 +10,9 @@ function test_build () {
     fi
 }
 
-test_build make ARCH=RV32 ocaml_emulator/riscv_ocaml_sim_RV32
-test_build make ARCH=RV64 ocaml_emulator/riscv_ocaml_sim_RV64
+test_build make ARCH=RV32 ocaml_emulator/riscv_ocaml_sim_RV32 -j24 
+test_build make ARCH=RV64 ocaml_emulator/riscv_ocaml_sim_RV64 -j24
 
-test_build make ARCH=RV32 c_emulator/riscv_sim_RV32
-test_build make ARCH=RV64 c_emulator/riscv_sim_RV64
+test_build make ARCH=RV32 c_emulator/riscv_sim_RV32 -j24
+test_build make ARCH=RV64 c_emulator/riscv_sim_RV64 -j24
 
```

Comment:
I dont see any reason that this is changed to exactly 24 jobs, one could argue that maybe should -j$(nproc), but I advise just to leave it as is here, unless we have a reason to change it !